llvm/lib/Transforms/IPO/SampleProfileMatcher.cpp

```diff
@@ -863,6 +883,39 @@ bool SampleProfileMatcher::functionMatchesProfile(Function &IRFunc,
   return Matched;
 }
 
+void SampleProfileMatcher::UpdateSampleLoaderWithRecoveredProfiles() {
+  DenseSet<StringRef> RecoveredFuncs;
```

Comment:
nit: we might want a better (and still concise) name, we don't recover functions..